File: cpp/chapter4/linearSearch.cpp
#include<iostream>
using namespace std;

int linearSearch(int a[9], int x){
    int n=9;

    for (int i = 0 ; i < n ; i++ ) {
    if ( a[i] == x ) return i;
    }
    return -1;
}
int main(){
int a[9]={1,3,4,4,6,17,79,81,90};
int x=79;


if (linearSearch(a,x) != -1)
    cout<<"The item was found"<<endl;
else
    cout<<"the item was not found"<<endl;

return 0;
}

File: python/chapter4/linearSearch.py

def linearSearch(a,x):
    n = len(a)
    for i in range(0,n):
        if a[i] == x:
            return i
    return -1

def main():
    a =[1,3,4,4,6,17,79,81,90]
    if linearSearch(a,90) != -1:
        print("item was found!")
    else:
        print("item was not found!")

main()